src/cli.js

```diff
@@ -206,12 +207,13 @@ function cli(inputArgs) {
         // calling into platform code should be dealing with this based
         // on the parsed args object.
         var downstreamArgs = [];
-        var argNames = [ 'debug', 'release', 'device', 'emulator', 'nobuild'];
+        var argNames = [ 'debug', 'release', 'device', 'emulator', 'nobuild', 'list'];
```

Comment:
Someone should split this array initialization line into multiple lines, one line per arg-name. But that doesn't have to be done in this commit.

(Splitting it means that you don't get blame for the entire line each time you add/remove an arg-name from the list, only for the one you're inserting/removing.)